@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -974,7 +981,7 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
                     ? featureHeight
                     : styleSetDefaultHeight !== undefined
                     ? styleSetDefaultHeight
-                    : 0;
+                    : MIN_BUILDING_HEIGHT;
```

Comment:
The `MIN_BUILDING_HEIGHT` won't be applied if theme will explicitly specify `height: 0`.
You should enforce `MIN_BUILDING_HEIGHT` after it has been calculated, not only change default.

Few lines below...
if (height < MIN_BUILDING_HEIGHT) { height = MIN_BUILDING_HEIGHT }

Reply:
After thinking about it, I think I'm gonna change the solution a bit, since current one will not cover tha case when ``height`` and ``floorHeight`` have the same value but are bigger than zero.